src/assets/dissolveMatchReasons.ts

```diff
@@ -6,7 +6,8 @@ export const studentReasonOptions = {
   5: 'Wir hatten Schwierigkeiten auf zwischenmenschlicher Ebene',
   6: 'Wir konnten keine gemeinsamen Termine finden',
   7: 'Wir hatten technische Schwierigkeiten',
-  8: 'Sonstiges',
+  8: 'Wir hatten sprachliche Schwierigkeiten',
```

Comment:
Don't we store the key only in the database? If yes `8` should definetly keep the value `'Sonstiges'` ...

Reply:
No, I don't think so. As you can see here:
https://github.com/corona-school/backend/blob/master/web/controllers/userController/index.ts#L956
the argument is passed as a string, and as far as I can tell, the only place where the deactivationReason is used, is in the transactionlog function in the last line of this snippet @Jonasdoubleyou 
https://github.com/corona-school/backend/blob/master/web/controllers/userController/index.ts#L988-L1014

Reply:
What you've linked is the "user deletes their account" endpoint, which also dissolves all the matches the user had (with reason 0). Users can also dissolve (aka "delete") a single match which is handled by [this endpoint](https://github.com/corona-school/backend/blob/4fe1375a3139ce95f8162e3cbb3bc27c36e06fce/web/controllers/matchController/index.ts#L117), and that stores the reason passed (as number) into the Match's field "dissolveReason". I would assume this endpoint is called by the logic you've touched ...

Reply:
I see, thanks for the clarification. Is there a frontend where these stats are shown? If so, I'll need to add a ninth reason there as well.

Reply:
Not that I'm aware of ... I guess we manually get such stats from the database